.github/workflows/update-sdk-versions.yaml

```diff
@@ -0,0 +1,160 @@
+          echo "Current iOS SDK version: $CURRENT_IOS"
+
+      - name: Get current Android SDK version
+        id: current-android
+        run: |
+          CURRENT_ANDROID=$(grep "android-client-sdk" android/build.gradle | sed -E 's/.*:([^"]+)".*/\1/')
+          echo "version=$CURRENT_ANDROID" >> $GITHUB_OUTPUT
+          echo "Current Android SDK version: $CURRENT_ANDROID"
+
+      - name: Check if updates are needed
+        id: check-updates
+        run: |
+          IOS_UPDATE_NEEDED=false
+          ANDROID_UPDATE_NEEDED=false
+
+          if [ "${{ steps.ios-version.outputs.version }}" != "${{ steps.current-ios.outputs.version }}" ]; then
+            IOS_UPDATE_NEEDED=true
+            echo "iOS SDK update needed: ${{ steps.current-ios.outputs.version }} -> ${{ steps.ios-version.outputs.version }}"
+          fi
+
+          if [ "${{ steps.android-version.outputs.version }}" != "${{ steps.current-android.outputs.version }}" ]; then
+            ANDROID_UPDATE_NEEDED=true
+            echo "Android SDK update needed: ${{ steps.current-android.outputs.version }} -> ${{ steps.android-version.outputs.version }}"
+          fi
+
+          if [ "$IOS_UPDATE_NEEDED" = true ] || [ "$ANDROID_UPDATE_NEEDED" = true ]; then
+            echo "updates-needed=true" >> $GITHUB_OUTPUT
+            echo "ios-update=$IOS_UPDATE_NEEDED" >> $GITHUB_OUTPUT
+            echo "android-update=$ANDROID_UPDATE_NEEDED" >> $GITHUB_OUTPUT
+          else
+            echo "updates-needed=false" >> $GITHUB_OUTPUT
+            echo "No SDK updates needed"
+          fi
+
+      - name: Update iOS SDK version
+        if: steps.check-updates.outputs.ios-update == 'true'
+        run: |
+          sed -i -E "s/(s\.dependency 'DevCycle', ').*(')/ \1${{ steps.ios-version.outputs.version }}\2/" ios/devcycle_flutter_client_sdk.podspec
+          echo "Updated iOS SDK version to ${{ steps.ios-version.outputs.version }}"
+
+      - name: Update Android SDK version
+        if: steps.check-updates.outputs.android-update == 'true'
+        run: |
+          sed -i -E 's/(implementation\("com\.devcycle:android-client-sdk:)[^"]+("\))/\1${{ steps.android-version.outputs.version }}\2/' android/build.gradle
+          echo "Updated Android SDK version to ${{ steps.android-version.outputs.version }}"
+
+      - name: Setup Ruby for CocoaPods
+        if: steps.check-updates.outputs.updates-needed == 'true'
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.0"
+          bundler-cache: true
+
+      - name: Validate iOS SDK version with CocoaPods
+        if: steps.check-updates.outputs.updates-needed == 'true'
+        run: |
+          cd example/ios
+          pod install --repo-update
+          echo "✅ iOS SDK version ${{ steps.ios-version.outputs.version }} resolved successfully"
+
+      - name: Setup Java for Gradle
+        if: steps.check-updates.outputs.updates-needed == 'true'
+        uses: actions/setup-java@v4
```

Comment:
vtagging here

Reply:
Those are internal GH ones, so I don't think they need it.